moped-editor/src/views/projects/projectView/ProjectComments.js

```diff
@@ -274,16 +266,17 @@ const ProjectComments = props => {
           {/*Now the notes*/}
           <Grid item xs={12}>
             <Card>
-              {data.moped_proj_notes.length > 0 ? (
+              {loading ? (
+                <CircularProgress />
+              ) : data?.moped_proj_notes?.length > 0 ? (
                 <List className={classes.root}>
                   {data.moped_proj_notes.map((item, i) => {
                     const isNotLastItem = i < data.moped_proj_notes.length - 1;
                     const editableComment =
                       userSessionData.user_id === item.added_by_user_id ||
                       userHighestRole === "moped-admin";
-
                     return (
-                      <>
+                      <React.Fragment key={item.project_note_id}>
```

Comment:
fixes a react warning about not having keys on list items

Reply:
Seems like a bit of code smell to have the `key` prop on `<React.Fragment>` instead of on `<ListItem>` component but I think I'm seeing why its necessary. I also have a slight preference for the `<>` fragment syntax so it feels like a step backward to have to spell it out. Is it because line 377 needs to stay inside the Fragment? Or could that line move outside the fragment but still inside the closing bracket on 379? Guess it doesn't really matter a whole bunch either way. Sorry for my rambling.